jenkins-pipelines/manager/ubuntu22-manager-backup-restore-custom-dataset.jenkinsfile

```diff
@@ -0,0 +1,17 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+managerPipeline(
+    backend: 'aws',
+    region: 'us-east-1',
+    test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_and_restore_only_data',
+    test_config: 'test-cases/manager/manager-backup-restore-set-dataset.yaml',
+    restore_params: "{'batch_size': 2, 'parallel': 1}",
```

Comment:
nitpick, the internal parameters in those are easy to get wrong.

might consider something similar to `SctKafkaConfiguration` and how it's uses pydantic 
to something that can be checked when sct config is being loaded.